exdb/views.py

```diff
@@ -150,9 +158,23 @@ def form_valid(self, form):
 
 
 class ViewExperienceView(TemplateView):
+    access_level = 'basic'
     template_name = 'exdb/experience_view.html'
 
     def get_context_data(self, **kwargs):
         context = super(ViewExperienceView, self).get_context_data()
         context['experience'] = get_object_or_404(Experience, pk=self.kwargs['pk'])
         return context
+
+
+class LoginView(TemplateView):
```

Comment:
You might look into using the Django's [default login view](https://docs.djangoproject.com/en/1.9/topics/auth/default/#django.contrib.auth.views.login) for this.